modeling.py

```diff
@@ -338,7 +338,8 @@ def get_assignment_map_from_checkpoint(tvars, init_checkpoint):
     (name, var) = (x[0], x[1])
     if name not in name_to_variable:
       continue
-    assignment_map[name] = name
+    #assignment_map[name] = name
+    assignment_map[name] = name_to_variable[name]
```

Comment:
unnecessary comment

Reply:
For non-trivial changes, I usually keep the old code in comment as reference. This can be removed later when all goes well.

This particular change is to deal with a problem that a perfectly running code (checkpoint loading) suddenly stopped to work for TF v2. I still haven't yet got time to fully understand why it failed for TF v2, and why that changed line makes it work. So I am keeping that old line in comment, and will come back to this later.